### PR TITLE
Joins

### PR DESCRIPTION
**Pull request by NurkaAmre**
### This PR I did following steps:
**- Created a table named owners**
sql1

**- Created a table named species**
sql2

**- Modified animals table add primary_key, deleted species added species_id & owner_id column**
sql3
sql5

**- Inserted data to owners table**
sql6

**- Inserted data to species table**
sql7

**- Fullfiled species_id column with value:**
sql8

**- Modified owner information** 
sql9

### - QUERIES
**- What animals belong to Melody Pond?**
sql10

**- List of all animals that are pokemon (their type is Pokemon).**
sql11

**- List all owners and their animals, remember to include those that don't own any animal.**
sql12

**- How many animals are there per species?**
sql13

**- List all Digimon owned by Jennifer Orwell.**
sql14

**- List all animals owned by Dean Winchester that haven't tried to escape.**
sql16

**- Who owns the most animals?**
sql16

### CODE REVIEW

**Review by codecaiine — COMMENTED** [review]
Hi @NurkaAmre, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the correct Readme Template. Please add the new descriptive README file to your project by using this [template](https://github.com/microverseinc/readme-template) and remember to customize it to your project and make sure that all the links in this readme work. :

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: data.sql [hunk 1]
```diff
@@ -30,4 +30,98 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+ON owners.id = animals.owner_id
+where owners.full_name = 'Dean Winchester' and animals.escape_attempts = 0;
+
+-- Who owns the most animals?
+SELECT owners.full_name, count(animals.id) AS animal_count
+FROM owners
+JOIN animals ON owners.id = animals.owner_id
+GROUP BY owners.full_name
+ORDER BY animal_count DESC
+LIMIT 1;
```
Comment (m05e5) [inline data.sql]: - [ ] Hi you really did a great job but as part of the requirements, you are supposed to create a `queries.sql` file and write your queries there

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @NurkaAmre , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @NurkaAmre, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**